### PR TITLE
Fixes #8488: Generate a custom index page for the manual

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/8488

### CODE REVIEW

**Comment by amousset** [comment]
Sill WIP, some issues to fix, but you can start reviewing the content/appearance of the page.
index

**Comment by amousset** [comment]
Commit modified

File: tools/generate_dynamic_content.py [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+  output.append("<h2>Resources</h2>")
+
+  output.append("""<p>Besides this documentation, you can find a lot of other useful information:</p>""")
+
+  output.append("<ul>")
+  output.append("<li>The <a href=\"http://www.rudder-project.org/changelog-" + version_info["version"] + "\">Changelog</a> for this version. It describes the changes in each release.</li>")
+  output.append("<li>The <a href=\"https://www.rudder-project.org/site\">Rudder project website</a>, particularly:")
+  output.append("<ul>")
+  output.append("<li>What is an ESR version? How long are Rudder versions supported? All the answers are in the <a href=\"https://www.rudder-project.org/site/documentation/faq/#versioning\">versionning policy</a>.")
```
Comment (jooooooon) [inline tools/generate_dynamic_content.py]: s/versionning/versioning/

File: tools/generate_dynamic_content.py [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+  output.append("</ul>")
+  output.append("<li>You need help or have questions about Rudder? You can:</li>")
+  output.append("<ul>")
+  output.append("<li>Send an email to the user <a href=\"https://www.rudder-project.org/site/support/forums-lists/\">mailing-list</a></li>")
+  output.append("<li>Join the <a href=\"https://www.rudder-project.org/site/support/chat-mailing-lists/\">IRC channel</a></li>")
+  output.append("</ul>")
+  output.append("<li>You need training,  or commercial support? Normation provides <a href=\"http://www.normation.com/en/rudder/services/\">services</a> for Rudder</li>")
+  output.append("<li>Browse the <a href=\"http://www.rudder-project.org/foswiki/Development/WebHome\">developper documentation</a> if you want to contribute or dive into Rudder internals</li>")
```
Comment (jooooooon) [inline tools/generate_dynamic_content.py]: s/developper/developer/

File: tools/generate_dynamic_content.py [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+
+  output.append("<p>If you are new to Rudder you can follow these steps to discover and start using Rudder:</p>")
+
+  output.append("<ul>")
+  output.append("<li>Read <a href=\"_introduction.html\">introduction</a> to get familiar with Rudder architecture and concepts.</li>")
+  output.append("""<li>Follow the <a href=\"_installation.html\">installation</a> section wich explains the <a href=\"configure_the_network.html\">requirements</a> and the installation steps for the different
+                 types of nodes:</li>""")
+  output.append("<ul>")
+  output.append("<li>The central <a href=\"_install_rudder_server.html\">server</a> </li>")
+  output.append("<li>The managed <a href=\"install-agent.html\">nodes</a></li>")
```
Comment (jooooooon) [inline tools/generate_dynamic_content.py]: Can we add an entry for relay servers here?

File: tools/generate_dynamic_content.py [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+  output.append("<li>What is an ESR version? How long are Rudder versions supported? All the answers are in the <a href=\"https://www.rudder-project.org/site/documentation/faq/#versioning\">versionning policy</a>.")
+  output.append("</ul>")
+  output.append("<li>You need help or have questions about Rudder? You can:</li>")
+  output.append("<ul>")
+  output.append("<li>Send an email to the user <a href=\"https://www.rudder-project.org/site/support/forums-lists/\">mailing-list</a></li>")
+  output.append("<li>Join the <a href=\"https://www.rudder-project.org/site/support/chat-mailing-lists/\">IRC channel</a></li>")
+  output.append("</ul>")
+  output.append("<li>You need training,  or commercial support? Normation provides <a href=\"http://www.normation.com/en/rudder/services/\">services</a> for Rudder</li>")
+  output.append("<li>Browse the <a href=\"http://www.rudder-project.org/foswiki/Development/WebHome\">developper documentation</a> if you want to contribute or dive into Rudder internals</li>")
+  output.append("</ul>")
```
Comment (jooooooon) [inline tools/generate_dynamic_content.py]: Does this rather large block of content have to in a Python file? Couldn't we have it elsewhere and include it at least?

File: tools/generate_dynamic_content.py (hunk 3 above)
Comment (amousset) [inline tools/generate_dynamic_content.py]: A plan to make another PR for 3.1, including relays in this list.

File: tools/generate_dynamic_content.py (hunk 4 above)
Comment (amousset) [inline tools/generate_dynamic_content.py]: It should probably be elsewhere, but we need some kind of templating.

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
OK, merging this PR
